composer.json

```diff
@@ -106,7 +106,7 @@
         "league/flysystem-sftp-v3": "^3.0",
         "mockery/mockery": "^1.5.1",
         "nyholm/psr7": "^1.2",
-        "orchestra/testbench-core": "^9.0",
+        "orchestra/testbench-core": "dev-persist-db-connections",
```

Comment:
@crynobone should we put this PR in draft while this dependency isn't ready yet?

Reply:
I can't merge Testbench Core branch to stable unless Laravel commit with this changes it Testbench will depends on the new `Illuminate\Foundation\Testing\DatabaseConnectionFactory` added in this PR.